--- file.cpp.src	2018-12-29 15:33:14.000000000 +0800
+++ file.cpp	2018-12-29 15:33:23.000000000 +0800
@@ -1573,13 +1573,13 @@ namespace libtorrent
 		}
 #endif
 
-#ifdef POSIX_FADV_RANDOM
-		if (mode & random_access)
-		{
-			// disable read-ahead
-			posix_fadvise(native_handle(), 0, 0, POSIX_FADV_RANDOM);
-		}
-#endif
+// #ifdef POSIX_FADV_RANDOM
+// 		if (mode & random_access)
+// 		{
+// 			// disable read-ahead
+// 			posix_fadvise(native_handle(), 0, 0, POSIX_FADV_RANDOM);
+// 		}
+// #endif
 
 #endif
 		m_open_mode = mode;
